app/model/DigitalEdition.scala

```diff
@@ -0,0 +1,25 @@
+package model
+
+import configuration.Config
+
+case class DigitalEdition(id: String, name: String, price: String, cmp: String)
```

Comment:
ha yeah I have no idea, I held off renaming it until I could find out
I shall do so though!